.github/workflows/dotnet-check.yml

```diff
@@ -13,6 +13,7 @@ permissions:
   contents: read
 jobs:
   dotnetcheck:
+    name: Lint DotNet
```

Comment:
@rlhagerm is this the right spelling of `(dot|\.)(net|NET)`?

Reply:
.NET is most correct, unless using the '.' causes issues, if so then dotnet is acceptable.

Reply:
if it's a big deal later on, we can change it pretty easily